### ISSUE TITLE
Update the GIF in the README.

### ISSUE THREAD

**Issue by mik01aj** [styleguidist/react-styleguidist#96]
We had a lot of changes in 2.0 and 2.1, it'd be cool to show them off. ;)

**Comment by sapegin** [comment]
Yeah, I need to do it. And maybe make a tiny (~1 minute) screencast or something.

**Comment by sapegin** [comment]
How about this? :-)

https://github.com/styleguidist/react-styleguidist/blob/next/Readme.md

**Comment by mik01aj** [comment]
Too fast or too choppy, it's hard to understand what's happening.

**Comment by sapegin** [comment]
I can increase the delay between frames. Or add more frames but it will make the file bigger.

**Comment by sapegin** [comment]
@mik01aj Now? 😊

**Comment by mik01aj** [comment]
Better, but still hard to follow. In particular, it's confusing when you switch views or files, there's not enough time to notice what you're clicking and have a look around. If you're worried about the file size, well, I'd say growing to 1MB would be still fine. And in any case, you could also reduce its pixel size since the viewport is 888px and your gif is 1206px wide.

**Comment by sapegin** [comment]
OK, I’ll try again later. Thanks for feedback!
